src/room/usePageUnload.ts

```diff
@@ -53,7 +39,7 @@ export function usePageUnload(callback: () => void) {
     }
 
     // iOS doesn't fire beforeunload event, so leave the call when you hide the page.
```

Comment:
```suggestion
    // iOS doesn't fire before unload event, so leave the call when you hide the page.
```
(not even part of your pr but worth changing?)

Reply:
This is how its spelt when adding the listener, so I'd argue this is how it should be referred to: https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event